recipes-app/iot2050-conf-webui/files/src/components/ExtendedModules/SM1223/SM1223_AC_DI8_DQ8RLY_Conf.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable camelcase */
+/* eslint-disable react/prop-types */
+import * as React from 'react';
+import SM1223Conf from './SM1223Conf';
+import uiString from '@/lib/uiString/SM1223.json';
```

Comment:
Imported but not used?

Reply:
Make sure no ESlint error during npm run build

Reply:
Done